Update user-permissions.md (#1551)
astronomer/docs: astro/user-permissions.md

:::info

> 1. If a user changes Workspace roles, it can take a maximum of 10 minutes for corresponding Airflow permission changes to take effect.

Reviewer: ```suggestion
When a user changes Workspace roles, it can take up to 10 minutes for the corresponding Airflow permissions to be updated.
```

> 2. There are three roles with respect to deployments - Deployment Admin, Deployment Editor and Deployment Viewer. This gives flexibilty to manage deployment level permissions along with Workspace level permissions. For example: One can have a view access at workspace level, yet have admin level access at airflow deployment level.

Reviewer: ```suggestion
There are three Deployment roles: Deployment Admin, Deployment Editor, and Deployment Viewer. These different roles let you manage Deployment and Workspace permissions individually. For example, a user can have Workspace view access and Deployment Admin permissions on a Deployment.
```

:::

:::caution

> If a user has Workspace admin role, irrespective of the Deployment role, they can perofrm admin level actions on the Airflow deployment.

Reviewer: ```suggestion
When a user is a Workspace admin, they assume Deployment Admin permissions on the Airflow deployment.
```
